### PR TITLE
ci: MariaDB needs at least 3 nodes to deploy Galera in Cloud 8

### PR DESCRIPTION
**Pull request by rsalevsky**

### CODE REVIEW

**Comment by cmurphy** [comment]
This still uses DRBD as the default storage mode, which can't run on three nodes: https://ci.suse.de/job/openstack-mkcloud/82069/parsed_console/

**Comment by jsuchome** [comment]
@cmurphy was this used in that build https://github.com/SUSE-Cloud/automation/pull/2175 ?

File: jenkins/ci.suse.de/openstack-mkcloud.yaml [hunk 1]
```diff
@@ -471,10 +468,21 @@
...
+                  nodes=4
+                  ;;
+            esac
+
+            : ${clusterconfig:=data+services+network=$clusternodes}
             export clusterconfig
-            if [[ $nodenumber -lt 3 ]] ; then
-              export nodenumber=3
+            if [[ $nodenumber -lt $nodes ]] ; then
+                export nodenumber=$nodes
```
Comment (dirkmueller) [inline jenkins/ci.suse.de/openstack-mkcloud.yaml]: not a big fan of silently messing with nodenumber.. we should error out in such mis cases.

**Comment by cmurphy** [comment]
@jsuchome looks like it, since this was in the console log:

```
++(mkcloud:192) show_environment(): git --no-pager show -s --oneline
+(mkcloud:192) show_environment(): echo '         git: 9f61c78e Merge pull request #2175 from jsuchome/mariadb-default'
         git: 9f61c78e Merge pull request #2175 from jsuchome/mariadb-default
```

**Comment by jsuchome** [comment]
Oh, @matelakat actually added another fix yesterday: https://github.com/SUSE-Cloud/automation/pull/2179

**Pull request merged**